java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/models/ShortenedStrings.java

```diff
@@ -385,19 +349,220 @@ else if (valuesField.type() instanceof ArrayType &&
         }
         return string;
     }
+    //would return char, but exceptions are expensive and so is boxing into
```

Comment:
Please move this line comment into the javadoc, if you want make it an implementation note, but there is no need for two comments here. Please also make sure, that there is an empty line between the end of the previous method and the comment.